src/tools/tool-options.h

```diff
@@ -82,7 +82,15 @@ struct ToolOptions : public Options {
       .addFeature(FeatureSet::MutableGlobals, "mutable globals")
       .addFeature(FeatureSet::TruncSat, "nontrapping float-to-int operations")
       .addFeature(FeatureSet::SIMD, "SIMD operations and types")
-      .addFeature(FeatureSet::BulkMemory, "bulk memory operations")
+      .addFeature(FeatureSet::BulkMemory,
+                  "bulk memory operations",
+                  FeatureSet(FeatureSet::BulkMemoryOpt))
+      .addFeature(FeatureSet::BulkMemoryOpt,
+                  "memory.copy and memory.fill",
+                  FeatureSet::None,
+                  FeatureSet(FeatureSet::BulkMemoryOpt))
+      .addFeature(FeatureSet::CallIndirectOverlong,
+                  "(ignored for compatibility)")
```

Comment:
Maybe we should write a little more here so the help message isn't "Enable (ignored for compatibility)" 😆

Reply:
Hm, yeah it's a little unfortunate that the "Enable" and "Disable" bits are generated separately... how about "Enable call-indirect-overlong (ignored for compatibility as this has no effect on Binaryen)" with the idea that call-indirect-overlong can now be looked up in tool-conventions along with the rest?

Reply:
sgtm 👍